modules/core/client/views/enroll.client.view.html

```diff
@@ -1,4 +1,5 @@
-<section data-ng-controller="EnrollController">
+<section data-ng-controller="EnrollController" id="content">
```

Comment:
!fix Ah, I see. You have a separate sidebar for each view. You need to find a way to get the sidebar into the index.html file in core>server, kinda like how the header is done to where it is the same one for each view and the views are changing but the header is static.